sdk/python/lib/pulumi/runtime/stack.py

```diff
@@ -169,10 +169,10 @@ def massage(attr: Any, seen: List[Any]):
     # make sure this is a popo.
     if isinstance(attr, dict):
         result = {}
-        for key, value in attr.items():
+        for key in attr:
```

Comment:
Add a comment on why we're not using `items()` here, so we don't accidentally regress in the future?